libs/internal/include/launchdarkly/events/client_events.hpp

```diff
@@ -1,50 +0,0 @@
-#pragma once
-
```

Comment:
These are no longer client-specific, so I've moved them to a `common_events.hpp`.